Orange/widgets/utils/owlearnerwidget.py

```diff
@@ -186,6 +186,9 @@ def check_data(self):
                 self.error(self.DATA_ERROR_ID,
                            "Data contains a single target value. "
                            "There is nothing to learn.")
+            elif self.data.X.size == 0:
+                self.error(self.DATA_ERROR_ID,
+                           "Found array with 0 feature(s) while a minimum of 1 is required.")
```

Comment:
"Data has no features to learn from."? Zero features are plural. :stuck_out_tongue: